conda-recipe/build.sh

```diff
@@ -29,7 +29,12 @@ $PYTHON setup.py build_clib
 $PYTHON setup.py build_ext install
 
 # Build wheel package
+if [ "$CONDA_PY" == "36" ]; then
+    WHEELS_BUILD_ARGS="-p manylinux1_x86_64"
+else
+    WHEELS_BUILD_ARGS="-p manylinux2014_x86_64"
+fi
 if [ -n "${WHEELS_OUTPUT_FOLDER}" ]; then
-    $PYTHON setup.py bdist_wheel -p manylinux1_x86_64
+    $PYTHON setup.py bdist_wheel ${WHEELS_BUILD_ARGS}
```

Comment:
need to discuss this offline. I do not understand why we need `wheels` in `conda build` and all these variables here.